connection-service/new-admin-vm/Install-ConnectionServer.ps1

```diff
@@ -227,26 +227,57 @@ Configuration InstallConnectionServer
             SetScript  = {
                 Write-Verbose "Install_SumoCollector"
 
-                $installerFileName = "SumoCollector.exe"
-                $sumo_package = 'https://collectors.sumologic.com/rest/download/win64'
-                $sumo_config = "$using:gitLocation/$using:sumoConf"
-                $sumo_collector_json = "$using:gitLocation/sumo-admin-vm.json"
                 $dest = "C:\sumo"
-                $destConf = "$dest\$using:sumoConf"
+                $installerFileName = "SumoCollector.exe"
 
-                Write-Host "Invoke-WebRequest -UseBasicParsing -Uri $sumo_config -PassThru -OutFile $destConf"
-                Invoke-WebRequest -UseBasicParsing -Uri $sumo_config -PassThru -OutFile $destConf
+                $sourceArray = @()
+                $destArray = @()
+
+                $sourceArray += "$using:gitLocation/$using:sumoConf"
+                $destArray += "$dest\$using:sumoConf"
+
+                $sourceArray += "$using:gitLocation/sumo-admin-vm.json"
+                $destArray += "$dest\sumo-admin-vm.json"
+
+                $sourceArray += "https://collectors.sumologic.com/rest/download/win64"
+                $destArray += "$dest\$installerFileName"
+
+                $orderNumArray = $using:orderNumArray
+                $retryMax = $using:retryCount
+                $downIdx = 0;
+                # sumological server require TLS 1.2
```

Comment:
Minor point: should be "sumologic" (here and in Install-PCoIPAgent.ps1) :)

Reply:
should be sumologic, I will change